.changeset/early-countries-sleep.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/clipboard-copy-library': major
+'@twilio-paste/core': minor
+---
+
+adds the clipboard-copy library; this will provide baseline functionality for developers to intuitively interact with the Clipboard API.
```

Comment:
didn't add too much tbh, i think the clipboard component will be more interesting than this one :-)